jobs/github/scripts/checkout.sh

```diff
@@ -53,7 +55,8 @@ if [ "$mergeable" != "true" ]; then
     exit 1
 fi
 
-merge_commit=$(jq -r ".\"merge_commit_sha\"" "$tmpname")
+MERGE_COMMIT=$(jq -r ".\"merge_commit_sha\"" "$tmpname")
+fi
```

Comment:
Is this just github not rendering whitespace, or do we need to indent this block